usaspending_api/references/tests/integration/test_load_rosetta.py

```diff
@@ -40,18 +40,18 @@ def test_rosetta_fresh_load():
             ]
         ],
         "headers": [
-            {"raw": "element", "display": "Element"},
-            {"raw": "definition", "display": "Definition"},
-            {"raw": "fpds_element", "display": "FPDS Data Dictionary Element"},
-            {"raw": "award_file", "display": "Award File"},
-            {"raw": "award_element", "display": "Award Element"},
-            {"raw": "subaward_file", "display": "Subaward File"},
-            {"raw": "subaward_element", "display": "Subaward Element"},
-            {"raw": "account_file", "display": "Account File"},
-            {"raw": "account_element", "display": "Account Element"},
-            {"raw": "legacy_award_file", "display": "Award File"},
-            {"raw": "legacy_award_element", "display": "Award Element"},
-            {"raw": "legacy_subaward_element", "display": "Subaward Element"},
+            {"raw": "A:element", "display": "Element"},
```

Comment:
What does the column name help with?

Reply:
uniqueness of the `raw` value. Frontend uses these to find the column to sort on.

Reply:
Previously the hard-coded list implicitly guarenteed this property, this PR changes it to be explicit.